test/jdk/java/net/httpclient/HttpResponseInputStreamInterruptTest.java

```diff
@@ -160,12 +163,11 @@ static Thread createClientThread(CountDownLatch countDownLatch, int port) {
                 assertEquals(firstOutput, FIRST_MESSAGE);
 
                 // countdown on latch, and assert that an IOException is throw due to the interrupt
-                countDownLatch.countDown();
-                assertThrows(IOException.class, () -> response.body().readAllBytes(), "excepted IOException");
+                interruptReadyLatch.countDown();
+                assertThrows(InterruptedIOException.class, () -> response.body().readAllBytes(), "excepted IOException");
```

Comment:
Maybe change the message while you're at it since we're actually expecting InterruptedIOException